apps/prairielearn/elements/pl-dropdown/pl-dropdown.py

```diff
@@ -58,7 +59,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     pl.check_attribs(
         element,
         required_attribs=["answers-name"],
-        optional_attribs=["blank", "allow-blank", "weight", "sort", "label"],
+        optional_attribs=["blank", "allow-blank", "weight", "sort", "aria-label"],
```

Comment:
Isn't this a breaking change? This element should accept `aria-label` in addition to `label` for backwards compatibility.

Reply:
I had the same thought, but this was added literally yesterday in #11844 and I haven't deployed yet, so I think this is safe. Ditto for the change to `pl-multiple-choice` below.

Reply:
Yep, that is from "I would also update elements I just changed for consistency."

Reply:
Ahhh ok, should be fine then 👍